### DISCUSSION TITLE
Move Copilot suggestions to Intellicode completion list

### DISCUSSION THREAD

**Discussion by TheGamer1002** [community/community#50918] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

I'd really like to see Copilot suggestions in the Intellicode menu in VS Code. It's a feature that's [present in my Neovim setup](https://github.com/zbirenbaum/copilot-cmp), and I think it's quite nifty. It also allows me to select other suggestions without the `Escape, Ctrl+Space` shortcut. Other than that, Copilot is integral to my IDE!

**Comment by iamcjwilliams** [comment]
@TheGamer1002 Thank you for your feedback, I will notify our copilot team 🚀

**Comment by iegik** [comment]
https://github.com/orgs/community/discussions/7510
https://github.com/github/docs/issues/19003

**Reply by TheGamer1002** [reply]
This talks about how there's conflicts between them, but no one's suggesting the two be integrated, which is what I'm looking for.
